crates/agglayer-certificate-orchestrator/src/error.rs

```diff
@@ -36,6 +37,10 @@ pub enum CertificationError {
     InternalError(String),
     #[error("Storage error: {0}")]
     Storage(#[from] agglayer_storage::error::Error),
+    #[error("rollup contract address not found")]
+    RollupContractAddressNotFound { source: L1RpcError },
+    #[error("Unable to find aggchain vkey")]
+    UnableToFindAggchainVkey { source: L1RpcError },
```

Comment:
```suggestion
    #[error("rollup contract address not found")]
    RollupContractAddressNotFound(#[source] L1RpcError),
    #[error("Unable to find aggchain vkey")]
    UnableToFindAggchainVkey(#[source] L1RpcError),
```

This'll make the `.map_err(|s| Error::Foo { s })` into the more pleasant `.map_err(Error::Foo)`